docs/tasks/semester-start-end-tasks.qmd

```diff
@@ -0,0 +1,49 @@
+---
+title: "DataHub Semester Start and End Tasks"
+---
+
+This document outlines the tasks for preparing DataHub for the start of a semester and for concluding semester activities.
+
+## Semester Start Tasks
+
+### 1. Setup and Configuration
+- [ ] **Update JupyterHub version**: Check for and apply updates to [Z2JH](https://z2jh.jupyter.org/en/latest/changelog.html) JupyterHub (if required)
+- [ ] **Update JupyterLab/Notebook version**: Check for and apply updates to JupyterLab and Notebook (if required)
+- [ ] **Bump Replica Values**: Bump replica values in values.yaml
+- [ ] **Set Node Count**: Set the appropriate number of node count for each node pool in GKE console
+- [ ] **Otter Grader Version Upgrade**: Check for and apply updates to Otter Grader across all hubs (if required)
+
+### 2. User Management
+- [ ] **Identify Unused Hubs**: Identify hubs that will not be used during a particular semester
+- [ ] **Send Onboarding Email**: Send onboarding instructions to all the instructors and GSIs added to `datahub-announce` listserv
+
+---
+
+## Semester End Tasks
+
+### 1. Operational Tasks
```

Comment:
add another task:  create a new github personal access token for our CI/CD pipeline (`DATAHUB_CREATE_PR`):  https://docs.datahub.berkeley.edu/admins/cicd-github-actions.html#organization-secrets

Reply:
Added